fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4

```diff
@@ -1429,7 +1429,7 @@ partitionValueList
     ;
 
 partitionValueDef
-    : INTEGER_VALUE | STRING_LITERAL | MAXVALUE | NULL
+    : SUBTRACT? INTEGER_VALUE | STRING_LITERAL | MAXVALUE | NULL
```

Comment:
default value for column has same problem, could u fix it togather?

Reply:
fixed